.env.example

```
@@ -0,0 +1,41 @@
+
+# LLMs
+ANTHROPIC_API_KEY=your_anthropic_key_here
+OPENAI_API_KEY=your_openai_key_here
+
+# Vector DB
+PINECONE_API_KEY=your_pinecone_key_here
+PINECONE_INDEX=tfl-monitor
+
+# LangSmith observability
+LANGSMITH_TRACING=true
+LANGSMITH_API_KEY=your_langsmith_key_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LANGSMITH_API_KEY key should go before the LANGSMITH_TRACING key

(UnorderedKey)